packages/pwa-kit-runtime/src/utils/loggerFactory.js

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * The PWAKITLogger provides structured logging with different log levels.
+ * @private
+ */
+export class PWAKITLogger {
```

Comment:
I think we predominantly use "PWAKit" (lowercase "it") vs. "PWAKIT". I'd recommend changing it for consistencies sake.